network/client.go

```diff
@@ -44,6 +44,7 @@ func (cl *Client) Send(dst *ServerIdentity, msg Body) (*Packet, error) {
 	baseIDLock.Unlock()
 	sid := NewServerIdentity(kp.Public, NewAddress(dst.Address.ConnType(),
 		"client:"+strconv.FormatUint(id, 10)))
+	sid.Cert = dst.Cert
```

Comment:
It makes it more easy to handle the `ServerIdentity` - if it gets stored in a configuration-file and used by another client, the cert is already there.